README.md

```diff
@@ -1,6 +1,10 @@
 # GOV.UK Connect
 
-This is a command line tool to help you connect to GOV.UK's infrastructure.
+This is a command line tool to help you connect to GOV.UK's infrastructure e.g.
+
+```bash
+govuk-connect ssh -e integration frontend
```

Comment:
```suggestion
gds govuk connect ssh -e integration frontend
```

One tool, then people don't get confused between whether they're SSHing or AWSing? Is it worth specifying the two commands?

Reply:
I think we should keep the guidance here self-contained, especially if someone doesn't have the GDS CLI installed (e.g. if they're not using AWS yet). Perhaps we can add another bit of guidance that talks about installing GDS CLI and subsequent usage?

Reply:
Fair enough, yeah. We can work on that guidance in the docs card in dev tools!